Version2

ryanmitchell opened the pull request:
Version 2 here we go

roydeanjr commented:
Did you update the marketplace?
I just looked at the marketplace version and it said 2.0.0 and that the extension was already installed on my install but it shows v1.1.0 or something like that.
I deleted the extension but not the data and now the extension will not install.
Any ideas?

ryanmitchell commented:
The marketplace updates automatically. I suspect it didnt install as your version was probably the v2 branch?

roydeanjr commented:
I reverted to an older version: v1.0.0 and the auto update signalled a new version was available, however, it reported that v1.1.0 was the new version available. I updated and that is for sure not version 2.0.0.

When I search for Kitchen in the Browse Extensions field I get that v2.0.0 is the version that is available. Yet, the system will not download and install v2.0.0. I even tried deleting the extension with delete data as the option and the progress bar displays, but does not install.

Nothing I do via the update process on the install will install v2.0.0.

roydeanjr commented:
![image](https://user-images.githubusercontent.com/36243458/100136445-ab3b8c80-2e50-11eb-83e1-60c63ef4b7dc.png)

roydeanjr commented:
![image](https://user-images.githubusercontent.com/36243458/100136517-c4dcd400-2e50-11eb-883b-8d2c57bab087.png)

ryanmitchell commented:
Just install by uploading this version. 



> On 24 Nov 2020, at 18:28, roydeanjr <notifications@github.com> wrote:
> 
> ﻿
> I reverted to an older version: v1.0.0 and the auto update signalled a new version was available, however, it reported that v1.1.0 was the new version available. I updated and that is for sure not version 2.0.0.
> 
> When I search for Kitchen in the Browse Extensions field I get that v2.0.0 is the version that is available. Yet, the system will not download and install v2.0.0. I even tried deleting the extension with delete data as the option and the progress bar displays, but does not install.
> 
> Nothing I do via the update process on the install will install v2.0.0.
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub, or unsubscribe.

ryanmitchell commented:
Oh wait. I didn’t tag the release - need to do that. 



> On 24 Nov 2020, at 18:28, roydeanjr <notifications@github.com> wrote:
> 
> ﻿
> I reverted to an older version: v1.0.0 and the auto update signalled a new version was available, however, it reported that v1.1.0 was the new version available. I updated and that is for sure not version 2.0.0.
> 
> When I search for Kitchen in the Browse Extensions field I get that v2.0.0 is the version that is available. Yet, the system will not download and install v2.0.0. I even tried deleting the extension with delete data as the option and the progress bar displays, but does not install.
> 
> Nothing I do via the update process on the install will install v2.0.0.
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub, or unsubscribe.

roydeanjr commented:
How long does it take to reflect on the marketplace?

ryanmitchell commented:
Seems to have updated there already.
https://tastyigniter.com/marketplace/item/thoughtco-kitchendisplay




> On 24 Nov 2020, at 18:55, roydeanjr <notifications@github.com> wrote:
> 
> 
> How long does it take to reflect on the marketplace?
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub <https://github.com/ryanmitchell/ti-ext-kitchendisplay/pull/24#issuecomment-733170187>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAAMVO2VYH3ZRAIBB5MGHOTSRP6SVANCNFSM4UBFE37Q>.
>

roydeanjr commented:
I attempted and it reflected a hash mismatch. I refreshed the updates screen and now it shows 2.0.0. Update completed. Thank you!

BreakSecurity commented on views/summary/view.blade.php:
```diff
@@ -26,6 +43,7 @@
 							<p class="w-100 text-wrap"><em>{!! $order->comment !}}</em></p>
```
{!! $order->comment !!}}

Missing !

roydeanjr replied:
Did not even notice that! And should have been caught when merged. Now fixed and a new PR created. Thank you!
